.storybook/preview.js

```diff
@@ -1,4 +1,16 @@
 
+import React from 'react'
+import GlobalStyles from 'styles/global'
+
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
-}
\ No newline at end of file
+}
+
+export const decorators = [
```

Comment:
nice! 

So this file still cannot easily be a TS? (I mean without nast workarounds)

Reply:
I just tried and turns out you can a23efe9